docs/samples/browser-plugin-meetings/app.js

```diff
@@ -710,9 +722,7 @@ const htmlMediaElements = [
 
 
 function cleanUpMedia() {
-  localMedia.microphoneStream = undefined;
-  localMedia.cameraStream = undefined;
-
+  // local streams can be used across meetings
```

Comment:
this is fine, but now there is no way to stop using the camera or mic while on the sample page. We probably need some UI for that

Reply:
Added 2 buttons to stop audio and video. Added listener for `stream-ended` when `.stop()` is called on stream.